redirect/routes.php

```diff
@@ -178,6 +178,10 @@ class Route {
             'description' => 'Grundlegende Algorithmen und Datenstrukturen',
             'target'      => 'http://campar.in.tum.de/Chair/TeachingSs18GAD',
         ],
+        'linalg'          => [
```

Comment:
the tag for 'Lineare Algebra für Informatik' is 'LA'

Reply:
What exactly do you mean? Are you suggesting an additional synonym?

Reply:
i don't know if it is possible otherwise i suggest to change the tag

Reply:
Sure is, checkout the synonyms array: https://github.com/phixxx5/TUM.sexy/blob/34391c0c78e5f87f0118833a4c367abf8c887213/redirect/routes.php#L255

Feel free to open a Pull Request for your suggestion :+1: